### DISCUSSION TITLE
Missing Armor Addon 1st Person Model File

### DISCUSSION THREAD

**Discussion by Noggog** [Mutagen-Modding/Mutagen.Bethesda.Analyzers#85] [Topic Codes]
# Context
Bethesda Path:
`ARMA/[MOD4/MOD5]`
Mutagen Path:
`ArmorAddon.WorldModel.[M/F].File`

# Description
Has a model listed, but the asset cannot be found in the load order

# Effects
Object is invisible

**Comment by Elscrux** [comment]
See this: https://ck.uesp.net/wiki/ArmorAddon
It is only an issue with both or only the female armor addon is assigned.
